config/initializers/rails_extensions.rb

```diff
@@ -0,0 +1,4 @@
+# recursive require of all files in lib/rails_extensions
+Dir[File.join(Rails.root, "lib", "rails_extensions", "**/*.rb")].each do |path|
+  require path
+end
```

Comment:
The convention of putting this in lib/rails_extensions and then having a special initializer is new to me. Looking at the current codebase, I've either done an initializer itself. Look at `extend_active_record.rb` (that probably deserves a more specific name) and `extend_anthropic_client.rb`. And I've also just re-opened the class itself within lib/ such as `lib/string.rb`. What do you think is the best pattern?

Actually, I just looked at the campfire codebase from 37signals and they do exactly what you just did. Interesting. I think this new pattern you're introducing might be the best pattern. Which means we'll just migrate the existing rails extensions over to this approach (in a different PR) and maybe those few ruby extensions should really have been initializers or maybe we also add a ruby_extensions equivalent? Not sure. All future PR. Nothing for you to do here but I'll just keep this comment since I thought out loud about it all. :)

Reply:
yeah I saw a few ways to do it when I was working on this. This way seemed to be the cleanest, but definitely can revisit if it starts to get messy